src/PerformanceTests/Common/Scenarios/SendLocalOneOnOne/SendLocalOneOnOneRunner.cs

```diff
@@ -16,9 +16,7 @@ partial class SendLocalOneOnOneRunner : BaseRunner
 
     protected override void Start()
```

Comment:
Because the code currently is pretty clean and adding async ceremony for micro-optimization is not needed.